crates/cli/src/opts/build/revive.rs

```diff
@@ -0,0 +1,60 @@
+use clap::Parser;
+use foundry_config::revive::ReviveConfig;
+use serde::Serialize;
+use std::path::PathBuf;
+
+#[derive(Clone, Debug, Default, Serialize, Parser)]
+#[clap(next_help_heading = "Revive configuration")]
+/// Compiler options for revive
+/// TODO: We need to add more revive specific arguments
+pub struct ReviveOpts {
+    #[clap(
+        value_name = "REVIVE_COMPILE",
+        help = "Enable compiling with revive",
+        long = "revive-compile",
+        action = clap::ArgAction::SetTrue,
+        default_value_t = false
+    )]
+    pub revive_compile: bool,
+
+    #[clap(help = "Solc path to be used by revive", long = "solc-path", value_name = "SOLC_PATH")]
```

Comment:
how solc is used in solidity compilation flow? why we need solc-path and they do not?

Reply:
I will actually remove this as we dont need it